components/gitpod-db/src/typeorm/entity/db-code-sync-resource.ts

```diff
@@ -17,14 +17,20 @@ export const enum SyncResource {
     Settings = 'settings',
     Keybindings = 'keybindings',
     Snippets = 'snippets',
+    Tasks = 'tasks',
     Extensions = 'extensions',
-    GlobalState = 'globalState'
+    GlobalState = 'globalState',
 }
-export const ALL_SYNC_RESOURCES: SyncResource[] = [SyncResource.Settings, SyncResource.Keybindings, SyncResource.Snippets, SyncResource.Extensions, SyncResource.GlobalState];
+export const ALL_SYNC_RESOURCES: SyncResource[] = [SyncResource.Settings, SyncResource.Keybindings, SyncResource.Snippets, SyncResource.Tasks, SyncResource.Extensions, SyncResource.GlobalState];
 
 export interface IUserDataManifest {
-    latest: Record<ServerResource, string>
-    session: string;
+	readonly latest?: Record<ServerResource, string>;
+	readonly session: string;
+    /**
+     * This property reflects a weak ETag for caching code sync responses,
+     * but this is not needed for the server, because it is the one which calculates it.
```

Comment:
```suggestion
     * this is not needed for the server as it's calculated by Express.js automatically.
```

Reply:
@jeanp413 I like the change but would make it not say Expres but rather "our backend" or "the server", because as I mentioned [in a previous comment](https://github.com/gitpod-io/gitpod/pull/8077#issuecomment-1047243114) we calculate it instead of Expres itself.

Reply:
Ah I missed that comment, I also forgot to mention that it's possible to override it, so in the case of `'/v1/resource/:resource/:ref'` endpoint we calculate it and in the case of `'/v1/manifest'` endpoint express does it